packages/flutter/lib/src/widgets/table.dart

```diff
@@ -340,7 +340,11 @@ class _TableElement extends RenderObjectElement {
     assert(renderObject != null);
     renderObject.setFlatChildren(
       _children.isNotEmpty ? _children[0].children.length : 0,
-      _children.expand((_TableElementRow row) => row.children.map((Element child) => child.renderObject)).toList()
+      _children.expand<RenderBox>((_TableElementRow row) =>
+          row.children.map<RenderBox>((Element child) {
```

Comment:
nit: use "return" instead of "=>" if it's going to wrap across multiple lines

Reply:
Done.